### PR TITLE
Buttons created

### PR DESCRIPTION
**Pull request by samscupoftea**
1. I added Buttons using discord.js module. 
2. Created button interactions using switch cases. So if you click on the button, it will tell you which prompt to type. 
3. I tried to get the button to trigger the corresponding code challenge but it kept crashing the bot.

### CODE REVIEW

File: bot.js [hunk 1]
```diff
@@ -1,14 +1,20 @@
...
 let conversation = [];
-let botIntroMsg = 'BOT INTRO MESSAGE! HARD CODED';
+let botIntroMsg = 'Hello there! I am a bot designed to help you practice your Javascript code-reading skills. I have 3 modes: Beginner, Intermediate and Advanced. Selecting one of these buttons will provide you with an appropriate Javascript example code to read. In addition to using the buttons, you can type prompts to start the challenges, too. For Beginner, type = !beginner | For Intermediate, type !intermediate | For Advanced, type !advanced.';
+
+const botButtons = [
+  { label: 'Beginner', customId: 'beginner' },
+  { label: 'Intermediate', customId: 'intermediate' },
+  { label: 'Advanced', customId: 'advanced' }
+];
 let beginner = "beginner";
```
Comment (josueJURE) [inline bot.js:14]: that line of code should perhaps be indented.

**Comment by josueJURE** [comment]
Excellent job, you're a developer in making 👍

**Comment by josueJURE** [comment]
Good work

**Comment by samscupoftea** [comment]
Thanks @josueJURE !!

**Pull request merged**